### PR TITLE
zh-cn: replace println with print since println is depreciated in Swift 2

### PR DESCRIPTION
**Pull request by major1xu**
This is corresponding to the following changes.
https://github.com/adambard/learnxinyminutes-docs/commit/775ac3b8595f80dd3229b7974672cf71d642d94f
and 
https://github.com/adambard/learnxinyminutes-docs/commit/c4558c47eaf08d7a1d22789f1b0a38c23260fd18

### CODE REVIEW

File: zh-cn/swift-cn.html.markdown [hunk 1]
```diff
@@ -28,7 +29,9 @@ import UIKit
 // TODO: TODO 标记
 // FIXME: FIXME 标记
 
-println("Hello, world")
+// 从 Swift 2 开始, print 和 print 合为 print method. Print 会自动加一个换行符.
```
Comment (geoffliu) [inline zh-cn/swift-cn.html.markdown]: print 和 print?

Comment (major1xu) [inline zh-cn/swift-cn.html.markdown]: Sorry, typo. Should be "println and print".

Comment (geoffliu) [inline zh-cn/swift-cn.html.markdown]: Ok, could you change it? Also, please resolve merge conflicts.

**Comment by geoffliu** [comment]
Looks like this is no longer necessary, it's already been done in 9b0242fbe95517db45347a0416b53e9ed6769bdd

**Pull request closed without merging**